app/models/school.rb

```diff
@@ -17,4 +17,24 @@ def full_address
     ADDRESS
     address.squeeze("\n")
   end
+
+  def fully_registered?
+    induction_coordinator_profiles
+      .joins(:user)
+      .where.not(users: { confirmed_at: nil })
+      .any?
+  end
+
+  def not_registered?
+    induction_coordinator_profiles.none?
+  end
+
+  def partially_registered?
+    return false if not_registered?
+
+    induction_coordinator_profiles
```

Comment:
This is just `!fully_registered?`, right?

Reply:
No, I think there's more to it than just !fully_registered? especially as we have 3 states i.e. not_registered? as well.

Reply:
Sorry, specifically lines 35-38

Reply:
The whole function is just `!(fully_registered? || not_registered?)` I believe

Reply:
Probably! More than happy to simplify it.